src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java

```diff
@@ -1054,14 +1054,12 @@ private String swapStore(String storeName, String directory) throws VoldemortExc
                                                                                                             .setStatus("started");
         try {
             if(!metadataStore.getReadOnlyFetchEnabledUnlocked()) {
-                throw new ReadOnlyFetchDisabledException("Read-only fetcher is disabled in "
-                                             + metadataStore.getServerStateUnlocked()
-                                             + " state on node " + metadataStore.getNodeId());
+                throw new ReadOnlyFetchDisabledException("Pushes to this node have been administratively disabled."
```

Comment:
You are right, BnP HA can trigger this as well. I think  hostname is already prefix on the BnP logging so id is a duplicate information.

When BnP HA is activated, the job usually succeeds, so we hardly see this error logs. This is important, when we are doing rolling upgrades when the job usually fails. For that use case, the new error message is better suited.